ceno_emul/src/platform.rs

```diff
@@ -10,8 +10,7 @@ use crate::addr::{Addr, RegIdx};
 #[derive(Clone, Debug)]
 pub struct Platform {
     pub rom: Range<Addr>,
-    // This is an `Option` to allow `const` here.
-    pub prog_data: Option<HashSet<Addr>>,
+    pub prog_data: BTreeSet<Addr>,
```

Comment:
`HashSet` doesn't have `const` in `pub fn new() -> HashSet<T, RandomState>`.

But `BTreeSet` has `pub const fn new() -> BTreeSet<T>`.